codemon/CodemonHelp.py

```diff
@@ -1,11 +1,29 @@
 from clint.textui import colored
+import os
 
 def showHelp():
-  print("           ---CODEMON---          ")
-  print(colored.cyan("  a CLI tool for competitive coders"))
-  print("\nCOMMANDS: \n")
-  print("codemon - - - - - - - - - - - - - - - shows help")
-  print("codemon init <contestName>  - - - - - initialises a contest")
-  print("codemon init -n <file>  - - - - - - - creates file with given name")
-  print("codemon listen  - - - - - - - - - - - compiles and gives output")
-  print("codemon practice <dirName>  - - - - - starts a practice session")
+  os.system('cls sys')
+  print(colored.green("           ---CODEMON---          "))
+  print("A CLI tool for competitive coders")
+  print("Get faster and more accurate during a coding contest.")
+  print("No more compile the code everytime you make a change. Just save the code and we will do the rest for you.")
+  print("Test your code with different inputs easily. Compile them locally. And, save those code files to your desired directory.")
+  print("Generate test cases and test those test cases with your code automatically.")
+  print(colored.cyan("\nQuick COMMANDS: \n"))
+  print(colored.yellow("codemon") + " - - - - - - - - - - - - - - -  - - A Overview of the program")
+  print(colored.yellow("codemon help") + " - - - - - - - - - - - - - - - Get answers to your common queries and list of all commands & features supported.")

```

Comment:
These two commands are repeated in the detailed function. I see no value in having two separate help functions. Let's remove this function and keep a single standard help function